### ISSUE TITLE
Character data

### ISSUE THREAD

**Issue by Alkahestry** [IAmPara0x/Yuno#5]
Hi, I can see that you did a great job on Yuno! As I'm on my way to figuring out how this model works and its preprocessing steps, I engaged some problems with the preprocessing part.

Currently I'm looking at the filter.py in the preprocessing folder. The class FilterText is initialized with AnimeInfo class.

```

class AnimeInfo(NamedTuple):

  uid: int

  names: List[str]

  characters: List[Character]

```

As I see, it contains anime uid(it should match MAL's uid), anime title and character info. But I couldn't find the character data in the Kaggle dataset or anime character scraper for MAL or anything. So that raised me some questions.

1. How did you attain the character data ? Is there any notable preprocessing needed to be done after fetching the data ?

2. What is included in the info for each anime character ? (their name, gender, age, ...etc)

By the way I think it'd be nice if you upload another notebook for the preprocessing part. You may be busy, so I don't mind if you don't. Thanks in advance!

**Comment by IAmPara0x** [comment]
The character data was taken from `Anilist`, and using their API. here's the notebook for that: https://www.kaggle.com/code/lollol222/get-data/notebook, actually it currently has some version issues. I will fix that later.

**Comment by Alkahestry** [comment]
Thanks for the reply, and sorry for rushing to close the ticket earlier. I also checked the issues, and saw some said the version compatible worked well in Python 3.6 instead of 3.7. Knowing that Kaggle's Python version is 3.7 and there's almost no way to change it without delving into the files from pc. I made the code available on Colab, there was no big issue and I got the file eventually. I'm moving on to the next part, thanks again!
